InternetHospital.BusinessLogic/Interfaces/IFilesService.cs

```diff
@@ -11,5 +12,9 @@ public interface IFilesService
         Task<User> UploadPassport(IFormFileCollection images, User user, DateTime addedTime);
         Task<User> UploadDiploma(IFormFileCollection images, User user, DateTime addedTime);
         Task<User> UploadLicense(IFormFileCollection images, User user, DateTime addedTime);
+
+        List<string> UploadArticleAttachment(IFormFile[] attachments, int articleId, bool isOnPreview);
```

Comment:
maybe its better to use more abstract container like IEnumerable?

Reply:
I use result in for cycle and need to have possibility to easy get item in specific position.